lib/fathead/htmlref/parse.py

```diff
@@ -31,8 +32,8 @@ def __init__(self, name, info, reference, example, see_also):
 
         # replace all newline and horizontal tab characters
         terms = {'\n': '\\\\n', '\t': '\\\\t', '\r': ''}
-        self.info = replace_all(self.info, terms)
-        self.example = replace_all(self.example, terms)
+        self.info = cgi.escape(replace_all(self.info, terms))
```

Comment:
@sriranganathan we actually do this internally -- but I'm having some issues with it not working for this particular Fathead. This is the bug I'm trying to fix ;)